docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -244,6 +294,52 @@
         @include hb-traditional {
           @include hb-pairing-color('background-color', 'tertiary-highlight');
         }
+
+        .hb-dark-pattern & {
+          @include hb-global-color('color', 'black');
+
+          & + .hb-nested-toggler,
+          &::after {
+            @include hb-icon-background('plus', 'black');
+          }
+
+          & + .hb-nested-toggler[aria-expanded="true"],
+          &[aria-expanded="true"]::after {
+            @include hb-icon-background('minus', 'black');
+          }
+        }
+      }
+
+      .hb-dark-pattern & {
+        @include hb-global-color('color', 'white');
+
+        &.is-active {
+          @include hb-global-color('color', 'black');
+          @include hb-pairing-color('background-color', 'secondary-highlight');
+
+          &::before {
+            @include hb-pairing-color('background-color', 'secondary');
+            content: '';
+            display: block;
+            width: hb-calculate-rems(5px);
+            position: absolute;
+            top: 0;
+            bottom: 0;
+            left: 0;
```

Comment:
We can remove the content, display, width, position, top, bottom, and left properties as they do not change from the values in `hb-main-nav__link--active-trial::before`.

![properties](https://user-images.githubusercontent.com/12678977/96596622-ea5b4880-12ba-11eb-9480-e02b33963cf3.png)

Reply:
This appears to be a local bug, not something I need to style for. This styling has been removed.